helm-hooks/preinstall/preinstall.go

```diff
@@ -49,24 +61,33 @@ func deleteExistingObjects() {
 }
 
 func createCertificates() {
-	cert := `{
+	certTmpl := Cert{namespace, serviceName, serviceNameDeployments}
+	tmpl := template.New("cert")
+	tmpl, err := tmpl.Parse(`{
 "hosts": [
-    "kritis-validation-hook",
-    "kritis-validation-hook.kube-system",
-    "kritis-validation-hook.%s",
-    "kritis-validation-hook.%s.svc",
-    "kritis-validation-hook-deployments",
-    "kritis-validation-hook-deployments.kube-system",
-    "kritis-validation-hook-deployments.%s",
-    "kritis-validation-hook-deployments.%s.svc"
+    "{{ .ServiceName }}",
```

Comment:
nit: pull this out to a constant

Reply:
And you can initialize it with "MustParse"

Reply:
I'm going to merge this now to get the parallel tests for other users and do this in another PR, what is MustParse in this context?  I don't see it in text/template